modules/API/Controller.cpp

```diff
@@ -0,0 +1,165 @@
+/**
+ * @file Controller.cpp
+ * @brief モーター制御に用いる関数をまとめたラッパークラス
+ * @author nishijima515
+ */
+#include "Controller.h"
+
+Controller::Controller()
+  : rightWheel(Port::PORT_A),
+    leftWheel(Port::PORT_B, Motor::EDirection::COUNTERCLOCKWISE),
+    armMotor(Port::PORT_C, Motor::EDirection::COUNTERCLOCKWISE)
+{
+}
+
+// power値の初期化
+int Controller::powerOfRightWheel = 0.0;
+int Controller::powerOfLeftWheel = 0.0;
+int Controller::powerOfArm = 0.0;
```

Comment:
```suggestion
int Controller::powerOfRightWheel = 0;
int Controller::powerOfLeftWheel = 0;
int Controller::powerOfArm = 0;
```
これもキャストされるからいいんだけども、int なら宣言で小数点以下はいらない。